tests/.env.test

```
@@ -1,33 +1,20 @@
+NEXT_PUBLIC_LIFI_API_URL=https://api-develop.jumper.exchange/pipeline
 NEXT_PUBLIC_WIDGET_INTEGRATOR=dev.jumper.exchange
 NEXT_PUBLIC_WIDGET_INTEGRATOR_REFUEL=dev.jumper.exchange.gas
 NEXT_PUBLIC_WIDGET_INTEGRATOR_BLOG=dev.jumper.exchange.blog
 NEXT_PUBLIC_INTEGRATOR_MOBILE=dev.jumper.exchange.m
 NEXT_PUBLIC_ADDRESSABLE_TID=0d48165d8b544e029b816f54c3371ae8
 NEXT_PUBLIC_CUSTOM_RPCS={ "59140": ["https://consensys-zkevm-goerli-prealpha.infura.io/v3/faf4bc4ea7344e5da5e56c55de087480"], "1151111081099710": ["https://chaotic-restless-putty.solana-mainnet.quiknode.pro/", "https://dacey-pp61jd-fast-mainnet.helius-rpc.com/"] }
 NEXT_PUBLIC_DKEY=AmE4rjRHH0aLksP_-Nym-DCL_5qOHBYR77m7svbGyHm5
-NEXT_PUBLIC_ONRAMPER_API_KEY=pk_prod_01GWH1ZXS9JS04JJQ5W4KWQBFF
-NEXT_PUBLIC_ONRAMPER_ENABLED=true
-NEXT_PUBLIC_BACKEND_URL=https://api-develop.jumper.exchange/v1
+NEXT_PUBLIC_BACKEND_URL=https://api-develop.jumper.exchange
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_BACKEND_URL key should go before the NEXT_PUBLIC_CUSTOM_RPCS key

(UnorderedKey)